### PR TITLE
[Added] Add certificate and private_key to link register and update

### PR DESCRIPTION
**Pull request by AlbertMaristany**

### CODE REVIEW

File: lib/belvo/resources.rb [hunk 1]
```diff
@@ -96,7 +99,9 @@ def register(
         token: options.token,
         encryption_key: options.encryption_key,
         access_mode: options.access_mode || AccessMode::SINGLE,
-        username_type: options.username_type
+        username_type: options.username_type,
+        certificate: Utils.read_file_to_b64(options.certificate),
```
Comment (menecio) [inline lib/belvo/resources.rb:103]: If you are doing it in options, you shouldn't need it here. Merge will take care of setting the value from options.

File: lib/belvo/utils.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require 'base64'
+
+class Utils
+  def self.read_file_to_b64(path)
+    puts path
+    unless path.nil?
+      if File.file?(path)
```
Comment (menecio) [inline lib/belvo/utils.rb:7]: can't we merge these into a single conditional?

File: lib/belvo/utils.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
```
Comment (menecio) [inline lib/belvo/utils.rb:1]: Can we have this class tested?

Comment (AlbertMaristany) [reply lib/belvo/utils.rb:1]: Added

**Pull request merged**